src/deepEqual.js

```diff
@@ -23,8 +23,55 @@
  *
  */
 
+function isEquals(value1, value2) {
+  if (typeof value1 === typeof value2) {
+    if (value1 === value2) {
+      return true;
+    }
+    return false;
+  }
+  return false;
+}
+
+function isObjectNotNull(elem, elem2) {
```

Comment:
based on the name it's not clear what function is doing. 

it's better to write function isSomething, than isNotSomething. then in code you can call

if (!isSomething(...)) - this is easy to understand. 
if (!isNotSomething(...)) - this is hard to understand.